packages/@react-spectrum/menu/chromatic/MenuTrigger.chromatic.tsx

```diff
@@ -143,6 +149,17 @@ const TemplateWithIcons = (): Story<SpectrumMenuTriggerProps> => (args) => (
   </MenuTrigger>
 );
 
+export const TemplateArabicWithIcons = (): Story<SpectrumMenuTriggerProps> => (args) => (
+  <MenuTrigger {...args} isOpen>
+    <ActionButton>
+      Menu Button
+    </ActionButton>
+    <Menu items={withArabic}>
+      {item => customMenuItem(item)}
+    </Menu>
+  </MenuTrigger>
+);
+
```

Comment:
This one isn't rendering in the MenuTrigger chromatic story. Instead of exporting this and importing it in the RTL chromatic story you could do the following:
```
const TemplateArabicWithIcons = (): Story<SpectrumMenuTriggerProps> => (args) => (
  <MenuTrigger {...args} isOpen>
    <ActionButton>
      Menu Button
    </ActionButton>
    <Menu items={withArabic}>
      {item => customMenuItem(item)}
    </Menu>
  </MenuTrigger>
);
...
export const TemplateArabic = TemplateArabicWithIcons().bind({});
TemplateArabic.storyName = 'Arabic complex items';
```
and in MenuTriggerRTL file:

```
export {
  TemplateArabic,
  Default,
  WithSections,
  Complex,
  AlignEnd,
  DirectionTop,
  DirectionBottom,
  DirectionStart,
  DirectionStartEnd,
  DirectionEnd,
  DirectionLeft,
  DirectionLeftEnd,
  DirectionRight,
  DirectionRightEnd
} from './MenuTrigger.chromatic';
```